plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/AddFileEntry.java

```diff
@@ -114,6 +114,9 @@ else if (stats.isPresent()) {
         this.parsedStats = resultParsedStats;
     }
 
+    /**
+     * paths are relative to the table location or absolute in case of shallow cloned table and are RFC 2396 URIs
```

Comment:
Either use `//` line comment or improve the javadoc.
If you decide for the latter, add `@see <a href="https://github.com/delta-io/delta/blob/master/PROTOCOL.md#add-file-and-remove-filel">Delta Lake protocol</a>

Reply:
this comment has not been handled.